app/src/main/java/org/sopt/sample/SignUpActivity.kt

```diff
@@ -0,0 +1,45 @@
+package org.sopt.sample
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.google.android.material.snackbar.Snackbar
+import org.sopt.sample.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivitySignUpBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        init()
+
+    }
+    fun init(){
+        binding.btnSignupFinish.setOnClickListener {
+            if(binding.etId.text.length < 6 || binding.etId.text.length > 10){
```

Comment:
```suggestion
            if(binding.etId.text.length <= 6 || binding.etId.text.length >= 10){
```
6자리, 10자리일 때도 포함해줘야 할 것 같아요!
`binding.etId.text.length in 6..10` 이렇게 `in`을 활용하면 가독성이 좋은 것 같아요! 저도 이렇게 고치려고요 🤭

Reply:
와! 코틀린 문법 공부를 해야할 것 같네요 ㅎㅎ 수정하겠습니다~